assets/src/edit-story/components/panels/design/textStyle/textStyle.js

```diff
@@ -30,16 +30,14 @@ import StyleControls from './style';
 import ColorControls from './color';
 import FontControls from './font';
 
-function StylePanel(props) {
-  const fontDropdownRef = useRef(null);
-  const textColorRef = useRef(null);
+const StylePanel = forwardRef(function StylePanel(props, ref) {
```

Comment:
Doesn't look like the highlighting is working as expected here (doesn't work on `main` either, probably due to the ref issues)

@BrittanyIRL Looks like the linking/highlighting from the pre-publish checklist is not working as expected for the text panel currently (e.g. contrast) -- is this something that is being addressed within the general pre-publish checklist work already?

Reply:
So these highlights are actually for the quick action menus that appear on the side of the canvas (see the gif I attached below). 

We're ripping up the old checklist and had to rewrite all of the highlights for the checklist 😓  so we'll be retesting those thoroughly, not worried about the old checklist.